examples/draggable-markers/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "scripts": {
+    "start": "webpack-dev-server --progress --hot --open",
+    "start-local": "webpack-dev-server --env.local --progress --hot --open"
+  },
+  "dependencies": {
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "react-map-gl": "^3.2.0"
+  },
+  "devDependencies": {
+    "babel-core": "^6.21.0",
+    "babel-loader": "^6.2.10",
+    "babel-preset-es2015": "^6.18.0",
+    "babel-preset-react": "^6.16.0",
+    "babel-preset-stage-2": "^6.18.0",
+    "webpack": "^2.4.0",
+    "webpack-dev-server": "^2.4.0"
+  }
+}
```

Comment:
We should eventually look into not having all this boilerplate for each example.

Reply:
> We should eventually look into not having all this boilerplate for each example.

It is our goal that each example folder can be independently copied out of this repo and work stand-alone, so we do want a minimal set of "boilerplate".

That said, we really want that boilerplate/configuration to be as minimal as possible.

Unfortunately the current fashion in javascript bundlers and compilers is to over-engineer for modularity and they do not seem to offer umbrella modules (not a single unneeded development plugin must installed, even if that requires the user to add 20 lines to package.json and has no effect the final bundle size :).

Two options here are probably

* move to `babel-preset-env` which should remove two lines
* explore `create-react-app`, which now seems to be supported by `mapbox-gl`